datamodel/initdb.d/05_reference_implementation_support.sql

```diff
@@ -52,6 +53,7 @@ UNION
     NULL::text AS equipment_reference,
     NULL::text AS empty_indicator_code,
     shipment_event.document_id AS document_id,
+    shipment_event.document_reference AS document_reference,

```

Comment:
does the document_reference column exist in the shipment_event table?

Reply:
Yes it does